src/ngraph/runtime/gpu/gpu_memory_manager.cpp

```diff
@@ -115,7 +115,7 @@ size_t runtime::gpu::GPUAllocator::reserve_argspace(const void* data, size_t siz
     // add parameter data to host buffer that will be transfered to device
     size = pass::MemoryManager::align(size, runtime::gpu::GPUMemoryManager::alignment);
     size_t offset = m_manager->queue_for_transfer(data, size);
-    auto local = --m_manager->m_argspace_mem.end();
+    auto local = std::prev(m_manager->m_argspace_mem.end());
     // return a lambda that will yield the gpu memory address. this
```

Comment:
Worth noting that `.rbegin()` would be nice here, however the standard does not guarantee reverse_iterators will remain stable upon insertion. It only makes that guarantee with forward iterators (c.f. https://stackoverflow.com/a/10086036)